modules/virt-setting-resource-quota-limits-for-vms.adoc

```diff
@@ -4,10 +4,45 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-setting-resource-quota-limits-for-vms_{context}"]
-= Setting resource quota limits for virtual machines
+= Enabling automatic resource quota limits for virtual machines
+
+When the `AutoResourceLimits` feature gate is enabled, {VirtProductName} automatically manages CPU and memory limits for virtual machines.
+
+By default, {VirtProductName} computes resource requests for a virtual machine. When you enable the `AutoResourceLimits` feature gate, {VirtProductName} also computes resource limits to meet namespace quota requirements.
+
+If a namespace enforces both CPU and memory quotas and requires limits to be set, enabling the `AutoResourceLimits` feature gate is recommended. When this feature is enabled, the memory limit is automatically set to double the base memory allocation and the CPU limit is set to one per vCPU. You can adjust the memory ratio per namespace by adding the `alpha.kubevirt.io/auto-memory-limits-ratio` label with a custom value. For example, setting `alpha.kubevirt.io/auto-memory-limits-ratio: 1.2` results in a memory limit equal to 1.2 times the memory request.
```

Comment:
> adding the `alpha.kubevirt.io/auto-memory-limits-ratio` label with a custom value

Where do we add this label? Perhaps show an example in the procedure with an Optional: callout?